comp/otelcol/status/component.go

```diff
@@ -0,0 +1,13 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+// Package status implements the core status component information provider interface
```

Comment:
Please follow the layout outline [here](https://datadoghq.dev/datadog-agent/components/overview/) for components.

Reply:
Updated my file structure for the latest component layout. I had to remove `fx.In` and `fx.Out` from my implementation file since it said that I couldn't import fx in the impl folder